docs/_posts/Meryem1425/2023-05-31-sbertresolve_icd10cm_slim_billable_hcc_en.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: model
+title: Sentence Entity Resolver for Billable ICD10-CM HCC Codes (sbertresolve_icd10cm_slim_billable_hcc)
+author: John Snow Labs
+name: sbertresolve_icd10cm_slim_billable_hcc
+date: 2023-05-31
+tags: [icd10cm, licensed, slim, en, clinical]
+task: Entity Resolution
+language: en
+edition: Healthcare NLP 4.4.2
+spark_version: 3.0
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps clinical entities and concepts to ICD-10-CM codes using sentence bert embeddings. In this model, synonyms having low cosine similarity to unnormalized terms are dropped. It also returns the official resolution text within the brackets inside the metadata. The model is augmented with synonyms, and previous augmentations are flexed according to cosine distances to unnormalized terms (ground truths).
+
+Outputs 7-digit billable ICD codes. In the result, look for aux_label parameter in the metadata to get HCC status. The HCC status can be divided to get further information: billable status, hcc status, and hcc score.For example, in the example shared below the billable status is 1, hcc status is 1, and hcc score is 11.
```

Comment:
lets share Hierarchical Condition Categories (HCC) when we mention for the first time.

Reply:
Done!